config/settings_schema.json

```diff
@@ -718,77 +718,141 @@
+        "label": "t:settings_schema.popups.settings.popup_shadow_opacity.label",
         "default": 0
       },
       {
         "type": "range",
-        "id": "popup_drawer_shadow_horizontal_offset",
+        "id": "popup_shadow_horizontal_offset",
         "min": -20,
         "max": 20,
         "step": 2,
         "unit": "px",
-        "label": "t:settings_schema.popup_drawer.settings.popup_drawer_shadow_horizontal_offset.label",
+        "label": "t:settings_schema.popups.settings.popup_shadow_horizontal_offset.label",
         "default": 0
       },
       {
         "type": "range",
-        "id": "popup_drawer_shadow_vertical_offset",
+        "id": "popup_shadow_vertical_offset",
         "min": -20,
         "max": 20,
         "step": 2,
         "unit": "px",
-        "label": "t:settings_schema.popup_drawer.settings.popup_drawer_shadow_vertical_offset.label",
+        "label": "t:settings_schema.popups.settings.popup_shadow_vertical_offset.label",
         "default": 0
       },
       {
         "type": "range",
-        "id": "popup_drawer_blur_radius",
+        "id": "popup_blur_radius",
         "min": 0,
         "max": 20,
         "step": 2,
         "unit": "px",
-        "label": "t:settings_schema.popup_drawer.settings.popup_drawer_blur_radius.label",
+        "label": "t:settings_schema.popups.settings.popup_blur_radius.label",
+        "default": 0
+      }
+    ]
+  },
+  {
+    "name": "t:settings_schema.drawers.name",
+    "settings": [
+      {
+        "type": "range",
+        "id": "drawer_border_width",
+        "min": 0,
+        "max": 10,
+        "step": 1,
+        "unit": "px",
+        "label": "t:settings_schema.drawers.settings.drawer_border_width.label",
+        "default": 1
+      },
+      {
+        "type": "range",
+        "id": "drawer_border_opacity",
+        "min": 0,
+        "max": 100,
+        "step": 5,
+        "unit": "%",
+        "label": "t:settings_schema.drawers.settings.drawer_border_opacity.label",
+        "default": 10
+      },
+      {
+        "type": "range",
+        "id": "drawer_shadow_opacity",
+        "min": 0,
+        "max": 100,
+        "step": 5,
+        "unit": "%",
+        "label": "t:settings_schema.drawers.settings.drawer_shadow_opacity.label",
+        "default": 0
+      },
+      {
+        "type": "range",
+        "id": "drawer_shadow_horizontal_offset",
+        "min": -20,
+        "max": 20,
+        "step": 2,
+        "unit": "px",
+        "label": "t:settings_schema.drawers.settings.drawer_shadow_horizontal_offset.label",
+        "default": 0
+      },
+      {
+        "type": "range",
+        "id": "drawer_shadow_vertical_offset",
+        "min": -20,
+        "max": 20,
+        "step": 2,
+        "unit": "px",
+        "label": "t:settings_schema.drawers.settings.drawer_shadow_vertical_offset.label",
+        "default": 0
+      },
+      {
+        "type": "range",
+        "id": "drawer_blur_radius",
```

Comment:
Minor, but would `drawer_shadow_blur_radius` be more consistent here?

Reply:
Yeah might as well fix it now. Updated.